leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java

```diff
@@ -632,6 +635,9 @@ private void getTransitionBundleParams(Bundle transitionBundle) {
     if (transitionBundle.keySet().contains(BACK_PRESSED_RETURN_OK)) {
       shouldReturnOkOnBackPressed = transitionBundle.getBoolean(BACK_PRESSED_RETURN_OK);
     }
+    if (transitionBundle.keySet().contains(ENABLE_SATELLITE_VIEW)) {
```

Comment:
Maybe, you will need to add this code also on the getSavedInstanceParams because if you change the orientation of the device you might loss the status of the button. Please verify it. Many thanks!